benchmark/bench-runner/Main.hs

```diff
@@ -74,7 +75,9 @@ rtsOpts exeName benchName0 = unwords [general, exeSpecific, benchSpecific]
         | "Data.Parser.ParserK/o-n-heap.someAlt" == benchName = "-K4M -M128M"
         | "Data.Parser.ParserK/o-n-heap.sequence" == benchName = "-M64M"
         | "Data.Parser.ParserK/o-n-heap.sequenceA" == benchName = "-M64M"
+#ifdef DEVBUILD
         | "Data.SmallArray.o-1-sp" `isPrefixOf` benchName = "-K128K"
+#endif
```

Comment:
This code will never be used as this file is never built with DEVBUILD flag. So better remove it than using it as dead code.